src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (type) {
```

Comment:
type of what?

Reply:
type of action...
<img width="821" alt="image" src="https://user-images.githubusercontent.com/122180564/219112028-8d086238-88ed-4871-8e28-5de874b08e47.png">